src/components/counter/Counter.jsx

```diff
@@ -0,0 +1,32 @@
+// Even met props notatie gedaan i.v.m. { fruitCount, setFruitCount } referentie.
+// fruitCount en setFruitCount lijken op placeholder State props, niet de daadwerkelijke state objecten zelf.
+// Op <Button /> component zit de daadwerkelijke onClick event handler.
+// Geen state op deze component pagina.
+// https://youtu.be/yvTGXH7uybA (Avoid this React State Mistake | React Previous State Explained)
+
+import './Counter.css';
+import Button from '../button/Button';
+
+export default function Counter(props) {
```

Comment:
Je kunt de props spreaden, zodat je niet over `props.` vóór hoeft te zetten
```suggestion
export default function Counter({ fruitCount, setFruitCount }) {
```